styles/header.css

```diff
@@ -53,18 +82,62 @@
   }
 }
 
+/* HRs in the navbar have to be positioned vertically relative to the nav lists,
+ * but horizontally relative to the entire container. They aren't valid as
+ * children of a <ul>, so they have to be siblings, and they can't be inside
+ * an <li> because they would be counted as items in the list. Repositioning of
+ * other elements during opening of the menu makes them jump around, so we also
+ * have to disable animation of opening/closing the nav. */
+.navbar hr {
+  opacity: 0.5;
+  border-color: var(--bs-navbar-color);
+  margin-left: calc(var(--bs-gutter-x) * -0.5);
+
+  .bg-dark & {
+    opacity: 0.75;
+  }
+
+  @media (width >= 576px) {
+    margin-left: calc((((100vw - 540px) / 2) * -1) - (var(--bs-gutter-x) * 0.5));
+  }
+}
+
+.navbar-nav.fw-bold {
```

Comment:
Why not just use `.fw-semibold` instead?

Reply:
Updated, thanks!